### DISCUSSION TITLE
Can't access funds from old Trinity wallet

### DISCUSSION THREAD

**Discussion by PrelawHalo** [iotaledger/firefly#7748] [Help & Support]
I am having trouble trying to move my funds from the old Trinity wallet (which is no longer active) to Firefly. I used this link to help (https://blog.iota.org/firefly-token-migration/), but a lot of what is pictured on this page doesn't show up in the Firefly wallet app. I have my seed saved from the Trinity wallet but Firefly never asks for this during the installation process in when creating a new account. Any help would be greatly appreciated.

**Comment by begonaalvarezd** [comment]
Hello, that post is deprecated.

The current ability to auto-migrate tokens from the pre-Chrysalis IOTA network via the Firefly wallet have been stopped with the IOTA Stardust upgrade. Once IOTA smart contracts are available on the IOTA network, this feature will be restored and migrations can be processed automatically again. 
We will provide a migration tool to resume legacy migrations, but its not finished yet

Will close the issue because Firefly is not intended to provide migration anymore, but it will be a different tool. The Foundation will notify it when the tool is ready 🌷

**Comment by PrelawHalo** [comment]
So there is currently no way for me to access those funds from the old wallet?

**Reply by begonaalvarezd** [reply]
No, you have to migrate your funds to the new network, the migration started on April 2021, but we recently did a new major upgrade to the network on October 4th and we are still in progress of updating the legacy migration process, once we release the new tool, I recommend you migrate your funds

**Reply by jklm2828** [reply]
Hello, 
i have the same problem!! How will we be updated on the release of the new tool? 

thanks you so much

**Reply by begonaalvarezd** [reply]
The Foundation will notify it when the tool is ready, i bet it will be shared in their classic channels like discord, the blog, twitter, ...

**Reply by jklm2828** [reply]
Hello,
i was searching for the social media channels. It seems like i'm too stupid to find them. Can you please let me know what the offical X (twitter) acc and discord is? 

Thanks for the help!

**Reply by begonaalvarezd** [reply]
you can see all the channels at the bottom of the official iota website https://www.iota.org/ 

![image](https://github.com/iotaledger/firefly/assets/3624944/6581132c-78a5-4cdf-8f5b-d758ee0d8991)


The blog is https://blog.iota.org/

**Reply by jklm2828** [reply]
Thank you. I'm following now! Do you know how long it takes till the tool is ready?
